### PR TITLE
UD-794 Add support for Che Admin plugins API

### PR DESCRIPTION
**Pull request by benoitf**
Change-Id: Ia96e4708729a1d05de18a56e046e04c91d4dd5fc
Signed-off-by: Florent BENOIT <fbenoit@codenvy.com>

### CODE REVIEW

**Comment by benoitf** [comment]
needs to wait server API implementation

File: src/app/admin/admin-config.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+/*
```
Comment (garagatyi) [inline src/app/admin/admin-config.js]: Do you know why copyright differs from copyright in java files?

Comment (benoitf) [inline src/app/admin/admin-config.js]: What differs ?

Comment (garagatyi) [inline src/app/admin/admin-config.js]: From java file
/*******************************************************************************
 * Copyright (c) 2012-2015 Codenvy, S.A.
 * All rights reserved. This program and the accompanying materials
 * are made available under the terms of the Eclipse Public License v1.0
 * which accompanies this distribution, and is available at
 * http://www.eclipse.org/legal/epl-v10.html
 *
 * Contributors:
 *   Codenvy, S.A. - initial API and implementation
 *******************************************************************************/

Comment (benoitf) [inline src/app/admin/admin-config.js]: If you're talking about stars http://code.mycila.com/license-maven-plugin/#license-templates and https://github.com/codenvy/codenvy-dev-resources/blob/master/codenvy-eclipse-license-resource-bundle/src/main/resources/javadoc.xml
JS have its own template

Comment (garagatyi) [inline src/app/admin/admin-config.js]: OK

**Comment by skabashnyuk** [comment]
LGTM

**Pull request merged**